package-system/OpenEXR/build_config.json

```diff
@@ -2,35 +2,64 @@
    "git_url":"https://github.com/AcademySoftwareFoundation/openexr.git",
    "git_tag":"v3.1.3",
    "package_name":"OpenEXR",
-   "package_version":"3.1.3-rev2",
+   "package_version":"3.1.3-rev3",
    "package_url":"https://www.openexr.com/",
    "package_license":"BSD-3-Clause",
    "package_license_file":"LICENSE.md",
    "cmake_find_source":"FindOpenEXR.cmake",
    "cmake_find_target":"FindOpenEXR.cmake",
    "extra_files_to_copy": [
-      ["FindImath.cmake", "FindImath.cmake"]
+      ["FindImath.cmake", "FindImath.cmake"],
+      ["../../Scripts/cmake/o3de_package_utilities.cmake", "o3de_package_utilities.cmake"]
+   ],
+   "cmake_install_filter":[
+      "*.cmake",
+      "*.a",
+      "*.lib",
+      "*.h",
+      "LICENSE.md"
+   ],
+   "cmake_generate_args_common" : [
+      "-DBUILD_TESTING=OFF",
+      "-DOPENEXR_BUILD_EXAMPLES=OFF",
+      "-DOPENEXR_INSTALL_EXAMPLES=OFF",
+      "-DOPENEXR_BUILD_TOOLS=OFF",
+      "-DOPENEXR_INSTALL_TOOLS=OFF",
+      "-DOPENEXR_RUN_FUZZ_TESTS=OFF",
+      "-DBUILD_SHARED_LIBS=OFF",
+      "-DDOCS=OFF",
+      "-DPYTHON=OFF"
    ],
    "Platforms":{
       "Windows":{
          "Windows":{
-            "build_configs" : ["debug", "release"],
+            "build_configs" : ["Debug", "Release"],
```

Comment:
Just wondering, does the case not matter? I noticed in visual studio we are using lower-case.

Reply:
I'll double check this, since I'm not sure why I changed it...  but bear in mind that these build configs are for the _package_ not for O3DE - as in, its the config to tell CMake to build the package source code that was downloaded from the official package site (like, from the OpenEXR homepage/git home).   Its very unlikely that for example OpenEXR comes with a (lowercase) "profile" or "debug" config, for example - it comes with the standard CMake ones (Release, Debug, RelWithDebInfo, and MinSizeRel).

Reply:
Yeah, this is correct, since its specifying the build configs to pass to cmake to build _the package itself_ from its original source code.  So it has to contain whatever the package expects (not what O3DE might expect)